tests/testthat/test-helpers_knitr.R

```diff
@@ -97,14 +100,20 @@ test_that("asis parameter works correctly for all implemented methods", {
       # If the default knit_print method has been overridden, test it
       if (h_custom_method_exists(knit_print, obj)) {
         rv <- knit_print(obj)
+        # Default behaviour
         expect_class(rv, "knit_asis")
+
+        # Explicit behaviours
         rv <- knit_print(obj, asis = TRUE)
         expect_class(rv, "knit_asis")
         rv <- knit_print(obj, asis = FALSE)
         # Most objects return a character, but not all.  For example,
         # CohortSizeDLT returns a knitr_table
         if ("knit_asis" %in% class(rv)) print(cls)
         expect_true(!("knit_asis" %in% class(rv)))
+
```

Comment:
are we sure we want to have all these html files in the snapshots? I have seen in the past that these can be very much depending on the system and package versions, so expect very frequent updates will be needed to avoid failing tests

Reply:
As we've discussed previously, snapshots aren't the ideal solution, but they are better than nothing.  In this case, I think they are slightly more relevant than usual as the purpose of `knit_print` is to provide a user-friendly rendition of a `crmPack` object.  If the irritation of maintaining the snapshots outweighs the effort of creating class-specific tests, I will reassess my position! :-)

Reply:
yeah I just remember what happened last time with snapshots - and html is even more fragile I think. if we can have your commitment to update the snapshots when it fails then ok